core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/groupBy.kt

```diff
@@ -48,21 +347,91 @@ public fun <T> DataFrame<T>.groupBy(vararg cols: AnyColumnReference, moveToTop:
 
 // endregion
 
+/**
+ * Groups the rows of this [Pivot] into [PivotGroupBy]
+ * based on the values in one or more specified [key columns][\columns].
+ *
+ * Works like regular [DataFrame.groupBy] on pivot groups.
+ *
+ * Grouping keys can also be created inline:
+ * ```kotlin
+ * // Create a new column "newName" based on existing "oldName" values
+ * // and use it as a grouping key:
+ * pivot.groupBy { expr("newName") { oldName.drop(5) } }
+ * ```
+ *
+ * @include [PivotGroupByDocs.CommonDescription]
+ */
+@ExcludeFromSources
+internal interface GroupByForPivotDocs
+
+/**
+ * {@include [GroupByForPivotDocs]}
+ * ### This `groupBy` Overload
+ */
+@ExcludeFromSources
+private interface CommonGroupByForPivotDocs
+
 // region Pivot
 
+/**
+ * {@include [CommonGroupByForPivotDocs]}
+ * @include [SelectingColumns.Dsl]
+ *
+ * #### For example:
+ *
+ * `pivot.`{@get [OPERATION]}` { length `[and][ColumnsSelectionDsl.and]` age }`
```

Comment:
why are you using `@get` here? They only serve purpose if these docs are included somewhere else and you need a different operation here. `[groupBy][groupBy]` is shorter to write even ;P